datajoint/heading.py

```diff
@@ -184,19 +191,19 @@ def init_from_database(self, conn, database, table_name):
                     attr['dtype'] = numeric_types[(t, is_unsigned)]
         self.attributes = OrderedDict([(q['name'], Attribute(**q)) for q in attributes])
 
-    def proj(self, *attribute_list, **renamed_attributes):
+    def proj(self, attribute_list, renamed_attributes, include_primary_key):
```

Comment:
Can we factor that into a function `_proj(attribute_list, renamed_attributes, include_primary_key)` and a function `proj(self, *attribute_list, **renamed_attributes)` that simply calls `_proj`? I really liked the old syntax of `project`. I can see why we want to have the other one, but I'd still like to keep the old syntax because

- it is significantly more effort to refactor old code (instead of simply renaming `project` into `proj`)
- it is a lot cleaner to call `rel.proj('myattr',new_attr='func(old_attr)')` instead of `rel.proj(['myattr'],dict(new_attr='func(old_attr)'})`